core/src/commonMain/kotlin/com/saveourtool/processbuilder/ExecutionResult.kt

```diff
@@ -0,0 +1,12 @@
+package com.saveourtool.processbuilder
+
+/**
+ * @property code exit code
+ * @property stdout content of stdout
+ * @property stderr content of stderr
+ */
+data class ExecutionResult(
+    val code: Int,
+    val stdout: List<String>,
```

Comment:
suggestion to store `stdout`/`stderr` as a single string instead of lines: it allows to store endline at the end of line.
we have some workaround for skipping last line in diktat testing. it was resolved by migration to single string

Reply:
And what's the problem with storing list of strings? Why do we need an endline?
We can either add the end of line we need just before printing or simply forget about the end of line and process the output as list of strings